site/docs/remote-caching.md

```diff
@@ -306,6 +307,31 @@ You may want to delete content from the cache to:
 * Create a clean cache after a cache was poisoned
 * Reduce the amount of storage used by deleting old outputs
 
+## Local action cache
+
+Bazel supports directory based action cache. It may be very useful, when
+working on multiple branches or multiple workspaces of the same project.
+It's also supported to provide user specific directory cache location, so
+that the local cache activation can be configured in project wide
+configuration file and committed into project repository. To activate the
+local action cache, add this line to your `tools/bazel.rc` file:
+
+```
+build --local_disk_cache=~/.gerrit/bazel-cache/cas
+```
+
+To have cache hits across different workspaces, additional option should be
+used to prevent Bazel to forward `PATH`, `LD_LIBRARY_PATH`, and `TMPDIR` env
```

Comment:
Why is this important for different workspaces? These environment variables should be the same for one machine and user? Are you thinking about the case where multiple users use the same cache?